app/views/keywords/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<a href="<%= keywords_path %>">
+  <%= I18n.t('keywords.to_home_page_link') %>
+</a>
+
+<div class="d-flex flex-column justify-content-center mt-4">
+  <p data-testid="keyword_details_content">Keyword: <%= keyword.content %></p>
+  <p data-testid="keyword_details_links_count">Total links: <%= keyword.links_count %></p>
+  <p data-testid="keyword_details_top_ads_count">Top ads count: <%= top_ads_count %></p>
+  <p data-testid="keyword_details_total_ads_count">Ads count: <%= total_ads_count %></p>
+  <p data-testid="keyword_details_non_ads_count">Non-ads count: <%= non_ads_count %></p>
```

Comment:
```suggestion
  <p data-testid="keyword_details_non_ads_count">I18n.t('xxx'): <%= non_ads_count %></p>
```

Reply:
Fixed in b99101be0f14e8d84cf2e88f9d68a836a980d681